python/cudf/cudf/utils/ioutils.py

```diff
@@ -176,6 +176,27 @@
 """
 doc_to_parquet = docfmt_partial(docstring=_docstring_to_parquet)
 
+_docstring_merge_parquet_filemetadata = """
+Write a DataFrame to the parquet format.
```

Comment:
This doesn't seem right for this function.

Reply:
Updated